packages/schemas/tables/sign_in_methods.sql

```diff
@@ -0,0 +1,5 @@
+create table sign_in_methods (
+  id varchar(128) not null,
+  name varchar(128) not null,
+  metadata jsonb /* @use SignInMethodMetadata */ not null default '{}'::jsonb
+)
```

Comment:
the trailing new line is expected

Reply:
updated add LOG-1418 issue to track the lint rule warning for this one